source/decisions/2023-09-25-support-djangocon-africa.rst

```diff
@@ -0,0 +1,18 @@
+2023-03-04 Support of DjangoCon Africa 2023
+===============================================
+
+Background
+----------
+
+DjangoCon Africa takes place on 6-11 of November 2023. On 7 September 2023 the DjangoCon
+Africa Chair contacted the trustees by email to request monetary support for the event.
+
+
+Resolution
+----------
+
+The decision was made to support DjangoCon Africa 2023 to a value equivalent to 
```

Comment:
I think this needs rewording slightly to match our previous resolution decisions:
```suggestion
To support DjangoCon Africa 2023 to a value equivalent to 
```